keras/utils/torch_utils.py

```diff
@@ -69,7 +71,13 @@ def call(self, inputs):
     ```
     """
 
-    def __init__(self, module, name=None):
+    def __init__(self, module, name="torch_module_wrapper"):
+        if name is None:
+            raise ValueError(
```

Comment:
`name` will be autogenerated and should not be required

Reply:
Please add a test with two or three `TorchModuleWrapper`'s. The name special casing is not needed.

Reply:
Added unit tests for multiple `TorchModuleWrapper` and also for tests for `autowrapping`.  The issue is, each of these models are independent `torch` models and have the same names for their sate variables.  This was the issue the name prefix was added to address. - 
```
(Pdb) model.torch_wrappers
[<TorchModuleWrapper name=torch_module_wrapper_6, built=True>, <TorchModuleWrapper name=torch_module_wrapper_7, built=True>]
(Pdb) model.torch_wrappers[0].state_dict()
OrderedDict([('module.0.weight', tensor([[ 0.4827, -0.3522],
        [-0.4961,  0.3488]])), ('module.0.bias', tensor([-0.1276, -0.0900]))])
(Pdb) model.torch_wrappers[1].state_dict()
OrderedDict([('module.0.weight', tensor([[-0.1081, -0.2072],
        [ 0.2882, -0.2535]])), ('module.0.bias', tensor([ 0.5350, -0.5053]))])
```

Reply:
Issue fixed.  I didn't have to save `_keys` or enforce names to be same.  It works well because `store` only gives its keys to the layer.  I didn't realize that.  I thought `store` gives ALL the keys of the model so there could be clashes.  Thanks for your patience in this review.